config/backend-config/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+resources:
+- backend-config-gsm.yaml
```

Comment:
Shouldn't we leave the dummy one uncommented by default?

Reply:
Yes we should